### ISSUE TITLE
User cannot access month selector buttons or year selector form in Date Picker Range Component

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1471]
## Environment
Chrome 70
macOS Mojave

## Violation
When the user activates the calendar drop down in the date picker component they cannot either 
1) user the arrow key down to select the left and right chevron to change the months and the year picker or 
2) tab down to the left and right chevrons and year picker in a logical order.

**Comment by SimonFinney** [comment]
What is the status of this issue?

**Comment by dakahn** [comment]
Issue is in our backlog and should be addressed this sprint. Feel free to submit a PR though! 👍

**Comment by dakahn** [comment]
When I was doing some accessibility testing about a month ago I opened [this issue](https://github.com/flatpickr/flatpickr/issues/1636
). As you can see the lack of keyboard navigation support is a known issue with the library we're using to display our calendar dropdown. I've spent the morning looking for a solution on our end, but it seems like we're stuck as long as we're using Flatpickr

Changing our Datepicker to use a [native html5 date input ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date)seems to be how to move forward. This is a larger -- and breaking change and will take more time.

**Comment by tw15egan** [comment]
Have we opened an issue over on Flatpickr?

**Comment by dakahn** [comment]
I opened an issue a month ago (linked above). Worth noting as well is that Flatpickr development [has slowed](https://github.com/flatpickr/flatpickr/issues/1646).

**Comment by dakahn** [comment]
I opened a ticket to cover the breaking changes proposed (in our React repo, since that's where I'm more comfortable working). I'm digging into this now and [you can follow progress there.](https://github.com/IBM/carbon-components-react/issues/1899).

**Comment by dakahn** [comment]
Leaving this issue open since these problems still exist. Datepicker is a refactor candidate for sure. Since we're moving forward with Flatpickr we need to find someway to allow the left and right arrows (changing the months) to be keyboard accessible. Doable, but would take some investigation.

**Comment by asudoh** [comment]
#3655 fixes the problem; Once you focus on date picker with down arrow key, you can use ctrl-left/right to go to previous/next month, ctrl-up/down to go to previous/next year.
